toontown/battle/BattleCalculatorAI.py

```diff
@@ -751,7 +751,7 @@ def __addDmgToBonuses(self, dmg, attackIndex, hp=1):
             return
         tgts = self.__createToonTargetList(toonId)
         for currTgt in tgts:
-            tgtPos = self.battle.suits.index(currTgt)
+            tgtPos = self.battle.activeSuits.index(currTgt)
```

Comment:
It would be nice to have a config setting that actually fixes the bug if enabled (by default).  People can turn the setting off if they want the true original TTO experience.

Reply:
true, but the config setting might cause client crashes

Reply:
This is an AI only change, I don't really see how it would crash the client.